lib/collective/collectors/honeybadger.rb

```diff
@@ -0,0 +1,62 @@
+module Collective::Collectors
+  class Honeybadger < Collective::Collector
+    requires :faraday
+    requires :faraday_middleware
+    requires :json
+
+    collect do
+      instrument_exceptions
+    end
+
+  private
+
+    def instrument_exceptions
+      each_result('/v1/projects') do |project|
+        id = project['id']
+        totals_by_env = Hash.new { |hash, key| hash[key] = 0 }
+
+        each_result("/v1/projects/#{id}/faults", resolved: 'f') do |fault|
+          totals_by_env[fault['environment']] += fault['notices_count']
+        end
+
+        group 'honeybadger' do |group|
+          totals_by_env.each do |env, total|
+          group.instrument 'faults.notices', total, source: "%s.%s" % [project['name'], env]
```

Comment:
I also found that the group call above will crash if the we don't call `group.instrument` within the block. e.g. if `totals_by_env` is empty. 

I'll fix this in `formatted_metrics`